runtime/src/bank.rs

```diff
@@ -314,6 +314,52 @@ impl CowCachedExecutors {
     }
 }
 
+mod executor_cache {
+    use super::*;
+
+    #[derive(Debug, Default)]
+    pub struct Stats {
+        pub hits: u64,
+        pub misses: u64,
+        pub evictions: HashMap<Pubkey, u64>,
+    }
+
+    impl Stats {
+        pub fn submit(&self, slot: Slot) {
+            let evictions: u64 = self.evictions.values().sum();
+            datapoint_info!(
+                "bank-executor-cache-stats",
+                ("slot", slot, i64),
+                ("hits", self.hits, i64),
+                ("misses", self.misses, i64),
+                ("evictions", evictions, i64),
+            );
+            debug!(
+                "Executor Cache Stats -- Hits: {}, Misses: {}, Evictions: {}",
+                self.hits, self.misses, evictions
+            );
+            if !self.evictions.is_empty() {
+                let mut evictions = self.evictions.iter().collect::<Vec<_>>();
```

Comment:
I hope this won't explode in size when running on the real deal.

Reply:
Yeah, same.  That's why they're trace logs instead of metrics.  Think we should cap it at something not too stupid?

Reply:
should we check to see if trace is enabled before doing the collecting/sorting work?

Reply:
> should we check to see if trace is enabled before doing the collecting/sorting work?

Yeah good call

Reply:
Done